src/ts/component/menu/smile.tsx

```diff
@@ -360,6 +365,12 @@ class MenuSmile extends React.Component<I.Menu, State> {
 			return (section.id == ID_RECENT) ? res : res + section.children.length; 
 		}, 0);
 
+		const fillRowWithBlank = (rowChildren) => {
+			if(rowChildren.length > 0 && rowChildren.length < LIMIT_ROW) {
```

Comment:
Add space after if, add parenthesises for condition statements (rowChildren.length > 0), move rowChildren.length to a variable